.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
```

Comment:
We can leave this false for now, but we should add another Issue for "Enable Style/Documentation Rubocop and update documentation accordingly". We've got some yard-style documentation, but it could be better.